metrics_sdk/lib/opentelemetry/sdk/metrics/instrument/histogram.rb

```diff
@@ -33,6 +31,12 @@ def record(amount, attributes: nil)
             OpenTelemetry.handle_error(exception: e)
```

Comment:
I can't comment on the line, but I think the call to `update` should change (the signature and other callers have changed).

Reply:
Updated 0dee2a29410ecee88b6c3569e9a2c65c63078e49